scripts/index.js

```diff
@@ -0,0 +1,364 @@
+//Apply Style based on screen Size.
+function loadDeviceStyle(href){
+  let link= document.createElement('link');
+  link.rel ="stylesheet";
+  link.href = href;
+  document.head.appendChild(link);
+}
+
+function applyDeciceStyle (){
```

Comment:
It is not recommended to use JS to load styles for different devices. Instead, it would be best to use [Media Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_media_queries/Using_media_queries), which is much more performant.

Reply:
removed and updated